src/template/src/fixtures/version.json

```diff
@@ -186,6 +190,12 @@
           {
             "fillType": "color",
             "blendMode": "normal",
+            "fill": {
+              "r": 0,
+              "g": 255,
+              "b": 255,
+              "a": 1
+            },
```

Comment:
`fill` field shouldn't be necessary in a fill object. Fill model uses `color` or `gradient` field based on the value of `fillType`.